conduit/lib/src/db/postgresql/postgresql_query.dart

```diff
@@ -10,18 +10,15 @@ class PostgresQuery<InstanceType extends ManagedObject> extends Object
     implements Query<InstanceType> {
   PostgresQuery(this.context);
 
-  PostgresQuery.withEntity(this.context, ManagedEntity? entity) {
-    _entity = entity;
-  }
+  PostgresQuery.withEntity(this.context, this._entity);
 
   @override
   ManagedContext context;
 
   @override
-  ManagedEntity? get entity =>
-      _entity ?? context.dataModel!.entityForType(InstanceType);
+  ManagedEntity get entity => _entity;
 
-  ManagedEntity? _entity;
+  late ManagedEntity _entity = context.dataModel!.entityForType(InstanceType);
```

Comment:
Is `late` necessary here?

Reply:
If you don't use `late` it will be always initialized to ` context.dataModel!.entityForType(InstanceType)` and the overwritten after that which is a waste.